.github/workflows/continuous-integration.yml

```diff
@@ -17,27 +17,23 @@ jobs:
         distribution: 'temurin'
         cache: maven
     - name: Checkout wala/IDE sources.
-      uses: actions/checkout@v3.5.2
-      with:
-        repository: wala/IDE
-        # fetch-depth: 50
-        path: ./IDE
+      run: git clone --depth=50 https://github.com/wala/IDE ${{ runner.temp }}/IDE
     - name: Checkout juliandolby/jython3 sources.
-      uses: actions/checkout@v3.5.2
-      with:
-        repository: juliandolby/jython3
-        path: ./jython3
+      run: git clone https://github.com/juliandolby/jython3.git ${{ runner.temp }}/jython3
```

Comment:
We could do `--depth=1` here too?

Reply:
See https://github.com/wala/ML/pull/61#discussion_r1239800169

Reply:
See https://github.com/wala/ML/pull/61#discussion_r1240026037 :-)